ocs_ci/ocs/resources/noobaa.py

```diff
@@ -48,16 +48,22 @@ def __init__(self):
             aws_secret_access_key=self.access_key
         )
 
-    def s3_create_bucket(self, bucketname):
+    def s3_create_bucket(self, bucketname, region='us-east-1'):
```

Comment:
The default region should be taken from the env vars:
```python
from ocs_ci.framework import config

self.region = config.ENV_DATA['region']
```

Reply:
@MeridianExplorer @parikshithb Please change the region here to be as written above, in a following PR.
Thanks

Reply:
Our main region is us-east-2 (Ohio).

Reply:
@RazTamir right!

Reply:
fixed

Reply:
need to re-set to region=config.ENV_DATA['region']